### ISSUE TITLE
Adjustable playback speed

### ISSUE THREAD

**Issue by mishari** [common-voice/common-voice#3116]
**Is your feature request related to a problem? Please describe.**
I find that my productivity would increase if I can adjust the audio playback speed. I usually listen to lectures at 1.25x playback speed, I think I can do the same for Common Voice. Other times I would like to slow down the speech on a particularly difficult sentence.

**Describe the solution you'd like**
Buttons or a slider that can help me adjust the speed. 0.5x - 1.25x at 0.25x increments. These are the options offered by Netflix and seem to be reasonable.

**Comment by darigovresearch** [comment]
Think this would be a helpful feature if easy to implement

**Comment by AigizK** [comment]
Yes, it makes checking very fast. I end up checking through an unofficial android application. They recently realized such an opportunity.

**Comment by darigovresearch** [comment]
@AigizK what's the application?

May be able to be ported if it is open source

**Comment by AigizK** [comment]
@darigovresearch "CV project" for android
